packages/@uppy/dashboard/src/components/FileCard/index.js

```diff
@@ -48,6 +45,17 @@ class FileCard extends Component {
     this.props.toggleFileCard(false)
   }
 
+  // TODO(aduh95): move this to `UNSAFE_componentWillMount` when updating to Preact X+.
```

Comment:
No other way apart from `UNSAFE_componentWillMount`? `constructor` not working for this?

Reply:
`constructor` is only called once (when the object is constructed), so if the object is unmounted then remounted, it would be problematic.
I would say this use case doesn't fall into the "unsafe" part of this API as we're not touching anything directly related with the rendering lifecycle, and that's why I picked it, but maybe I'm missing something.

Reply:
I’m mostly concerned about this being removed in future versions of Preact.